source/server/options_impl.cc

```diff
@@ -144,6 +150,6 @@ OptionsImpl::OptionsImpl(int argc, char** argv, const HotRestartVersionCb& hot_r
   drain_time_ = std::chrono::seconds(drain_time_s.getValue());
   parent_shutdown_time_ = std::chrono::seconds(parent_shutdown_time_s.getValue());
   max_stats_ = max_stats.getValue();
-  max_stat_name_length_ = max_stat_name_len.getValue();
+  max_stat_name_length_ = max_obj_name_len.getValue() + ENVOY_MAX_STAT_SUFFIX_LENGTH;
```

Comment:
The user-specified option is now max_stat_name_len.  For consistency, please replace the method  Options::maxStatNameLength() with Options::maxObjNameLen()

Reply:
I intentionally left it as it is. Replacing it with maxObjectNameLen will require each call site to use maxObjectNameLen + maxStatSuffixLen. That seems unnecessary detail across the code base where maxStatNameLen is used.

Reply:
There should only end up being one call-site: RawStatData::configure().  And then there will be 2 static functions on RawStatData; maxNameLength() and maxObjNameLength(), one of which will be implemented in terms of the other (adding or subtracting 67).